reference/configuration/framework.rst

```diff
@@ -1187,7 +1189,100 @@ individually for each asset package:
 
 .. note::
 
-    This parameter cannot be set at the same time as ``version``.
+    This parameter cannot be set at the same time as ``version`` or ``json_manifest_path``.
+
+.. _reference-assets-json-manifest-path:
+.. _reference-templating-json-manifest-path:
+
+json_manifest_path
+..................
+
+**type**: ``string`` **default**: ``null``
+
+.. versionadded:: 3.3
+
+    The ``json_manifest_path`` option was introduced in Symfony 3.3.
+
+The file path to a ``manifest.json`` file containing an associative array of asset
+names and their respective compiled names. A common cache-busting technique using
+a ``manifest.json`` file works by writing out assets with a "hash" appended to their
+file names (e.g. ``main.ae433f1cb.css``). You can incorporate this into many work
+flows, including Webpack and Gulp using `webpack-manifest-plugin`_ and `gulp-rev`_,
```

Comment:
It does: added a note about that in combination with references for straight-webpack and gulp.